src/components/calcite-slider/calcite-slider.tsx

```diff
@@ -825,7 +911,13 @@ export class CalciteSlider {
     let current = this.min;
     while (this.ticks && current < this.max + this.ticks) {
       ticks.push(current);
-      current = current + this.ticks;
+      // match tick label formatting; handle binary floating point numbers not mapping correctly to decimals (precision loss with `number` type)
+      // https://v8.dev/features/bigint
+      const currentWithTick = (current + this.ticks).toLocaleString();
```

Comment:
Assuming `this.ticks` always contains non-localized numbers, then we probably shouldn't convert ☝️ this value to a locale string to immediately construct it back to a `Number` here 👇 .  The locale string versions of the numbers should only be used for display.

Reply:
Got it, I'll remove 👍